keyserver/listener.go

```diff
@@ -100,6 +100,37 @@ func (server *ConiksServer) handleClientMessage(msg []byte) ([]byte, error) {
 
 	// handle request
 	switch req.Type {
+	default:
+		log.Printf("unknown message type: %q", req.Type)
+		response = p.NewErrorResponse(p.ErrorMalformedClientMessage)
+		err = p.Error(p.ErrorMalformedClientMessage)
+	}
+
+marshalling:
+	res, e := json.Marshal(response)
+	if e != nil {
+		panic(p.Error(p.ErrorInternalServer))
+	}
+	return res, err
+}
+
+func (server *ConiksServer) handleBotMessage(msg []byte) ([]byte, error) {
```

Comment:
No worries if you don't have a solution for this yet. This isn't super urgent, but it would be nice to refactor this as this will be useful if/when we need to write additional handlers.